ViAn/GUI/Analysis/analysisslider.cpp

```diff
@@ -31,11 +31,19 @@ void AnalysisSlider::paintEvent(QPaintEvent *ev) {
     //Get one frame's width of the slider
     double c = (double)(groove_rect.right()-groove_rect.left())/maximum();
```

Comment:
Should have a more descriptive name than `c`. Perhaps `frame_width` or similar. I know that this wasn't a new addition but we should still change it.